### DISCUSSION TITLE
Show tray icons on additional monitors?

### DISCUSSION THREAD

**Discussion by RChesterton** [Rem0o/FanControl.Releases#2008] [Ideas]
It would be great if we could see the selected tray icons on second/third monitors.

Right now, they only display on my primary monitor and so can't be seen when a game is running full screen.

**Comment by aleggst** [comment]
This would be a much appreciated feature from my side as well. (in case this cannot be implemented due to Windows Limitations, some sort of "mini/slim" window mode could be a workaround.)
